discovery/README.md

```diff
@@ -12,6 +12,9 @@ Docker Swarm comes with multiple Discovery backends.
 
 ### Hosted Discovery with Docker Hub
 
+This service is hosted by Docker Hub and requires all nodes to have
```

Comment:
The "service" referred to here is the "Hosted Discovery (service)", which is hosted on Docker Hub (but perhaps "Docker Hub" should be left out here, because that's usually associated with the Docker Hub *Registry*)

@vieux is there an "official" / vetted name for the Discovery Service?